check_whitespace.c

```diff
@@ -66,7 +67,10 @@ int is_clean(char* str) {
   // 0 if they're equal, and a positive value if the first is
   // greater than the second.
   result = strcmp(str, cleaned);
-
+  //free(cleaned);
```

Comment:
I would remove code that's commented out like this. You don't want to collect the history of all your good (& bad) ideas in comments like this.
```suggestion
```